src/client/refresh/gl1/gl1_draw.c

```diff
@@ -330,7 +330,7 @@ RDraw_FadeScreen(void)
 }
 
 void
-RDraw_StretchRaw(int x, int y, int w, int h, int cols, int rows, byte *data)
+RDraw_StretchRaw(int x, int y, int w, int h, int cols, int rows, const byte *data, int bits)
```

Comment:
Declaring 'data' as const leads to a new warning further down below:

```
src/client/refresh/gl1/gl1_draw.c: In function 'RDraw_StretchRaw':
src/client/refresh/gl1/gl1_draw.c:451:40: warning: assignment discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
  451 |                                 source = data + cols * row;
      |                                        ^
src/client/refresh/gl1/gl1_draw.c:482:32: warning: assignment discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
  482 |                         source = data + cols * row;
      |
```

Reply:
Thank you! Fixed!